### ISSUE TITLE
Update advice on using pseudonyms to suggest not using an obvious one!

### ISSUE THREAD

**Issue by RichardTaylor** [mysociety/whatdotheyknow-theme#1211]
Draft additional text for https://www.whatdotheyknow.com/help/privacy#real_name

>If you are going to use a pseudonym, try to pick something which looks like a regular name. Try to resist the urge to be funny or clever when deciding on a pseudonym. Often we see requests being rejected on the grounds of an obvious pseudonym having been used, but if a plain name had been used the information requested would probably have been released. 

I'm just noting, and developing, an idea raised on Slack here.

**Comment by confirmordeny** [comment]
I believe there is a tendency to:
(a) select a name that is alliterative e.g. Sam Smith; and/or
(b) pick a family name that in some way resembles a first name e.g. Jackson, Johnson, Richards.

Also, worth reminding people that you don't need to give a first name: "Mr Brown" and "Mrs Wilson" are both perfectly valid.

**Comment by RichardTaylor** [comment]
We could link to one or more random name generators eg. 

https://www.behindthename.com/random/random.php

That one can even adjusted to offer English/Welsh/Scottish/Irish names 

https://www.behindthename.com/random/random.php?gender=both&number=2&sets=1&surname=&usage_eng=1&usage_iri=1&usage_sco=1&usage_wel=1

There is an Alavetli ticket for a integrating a pseudonym generator into the service https://github.com/mysociety/alaveteli/issues/6174 - linking to one or more of them would be a low level of integration.

**Comment by RichardTaylor** [comment]
Those using a non-obvious pseudonym may even be able to refer a case to the ICO under that pseudonym. 

We already advise that one risk of using a pseudonym is that it might make it harder to seek enforcement action from the ICO, or appeal beyond that stage.

**Comment by HelenWDTK** [comment]
This issue is being closed due to a lack of discussion or resolution for over 12 months. Should we decide to revisit this issue in the future, it can be reopened.
